### PR TITLE
feat(components): export banner type

### PR DESCRIPTION
**Pull request by deepvishwas**
## Motivations

In order to integrate the Banner component, we need the banner type to be exported. 

### Added

- 

### Changed

- 

### Deprecated

No banner type is exported

### Fixed

Banner type is exported

### Security

- 

## Testing

Banner type should be available to import 
---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

Random photo of Atlantis

### CODE REVIEW

File: packages/components/src/Banner/Banner.tsx [hunk 1]
```diff
@@ -8,9 +8,10 @@ import { Icon } from "../Icon";
 import { Text } from "../Text";
 import { Button, ButtonProps } from "../Button";
 
+export type BannerType = "notice" | "success" | "warning" | "error";
```
Comment (eddysims) [inline packages/components/src/Banner/Banner.tsx:11]: Can we add a white space line here.

File: packages/components/src/Banner/Banner.mdx [hunk 2]
```diff
@@ -30,7 +30,8 @@ import { Banner } from "@jobber/components/Banner";
 
 <Props of={Banner} />
 
----
+Banner type should be one of the following "notice" | "success" | "warning" |
+"error".
```
Comment (eddysims) [inline packages/components/src/Banner/Banner.mdx:34]: For this, we have the different `types` below in the `mdx`. We may want to change this to be a more readable header with sub-headings for each type.

Thoughts @chris-at-jobber

**Review by eddysims — COMMENTED** [review]
@deepvishwas a few thoughts here

File: packages/components/src/Banner/Banner.mdx (hunk 2 above)
Comment (deepvishwas) [reply packages/components/src/Banner/Banner.mdx:34]: How about this 
image

File: packages/components/src/Banner/Banner.tsx (hunk 1 above)
Comment (deepvishwas) [reply packages/components/src/Banner/Banner.tsx:11]: 👍🏻

File: packages/components/src/Banner/Banner.mdx (hunk 2 above)
Comment (chris-at-jobber) [reply packages/components/src/Banner/Banner.mdx]: 👍  basically just pushed this up but with links to each type's usage guidelines

Comment (chris-at-jobber) [reply packages/components/src/Banner/Banner.mdx]: Also nests the usage guidelines for the respective types under this new header!

Comment (deepvishwas) [reply packages/components/src/Banner/Banner.mdx]: @chris-at-jobber  Done! I've added links too.

**Review by eddysims — APPROVED** [review]

**Pull request merged**